subgraphs/reviews/reviews.graphql

```diff
@@ -0,0 +1,36 @@
+schema @link(url: "https://specs.apollo.dev/federation/v2.0", import: ["@key", "@override", "@shareable"]) {
+  query: Query
+}
+
+type Product implements ProductItf @key(fields: "id") {
+  id: ID!
+  reviewsCount: Int!
+  reviewsScore: Float! @shareable @override(from: "products")
```

Comment:
it might be simpler to just show `@override` without the `@shareable`, but this definitely works!

Reply:
done!